client/src/components/Workflow/Editor/Attributes.vue

```diff
@@ -181,6 +181,10 @@ export default {
         onTags(tags) {
             this.tagsCurrent = tags;
             this.onAttributes({ tags });
+            this.$emit("input", this.tagsCurrent);
+        },
+        onTagClick(tag) {
+            this.$emit("tag-click", tag);
         },
```

Comment:
This event seems to be consumed nowhere. If I missed something and it is, `StatelessTags` should be set to clickable.

Reply:
@ElectronicBlueberry I didn't think we needed the extra click/link so I removed it here.